src/fsharp/FSharp.Build/Microsoft.NET.Sdk.FSharp.props

```diff
@@ -0,0 +1,26 @@
+<!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.FSharp.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(MSBuildRuntimeType)' == 'core' ">
+    <FscToolPath Condition="'$(FscToolPath)' == ''">$(MSBuildThisFileDirectory)</FscToolPath>
+    <FscToolExe Condition="'$(OS)' != 'Unix' and '$(FscToolExe)' == ''">RunFsc.cmd</FscToolExe>
```

Comment:
Where/what are ``RunFcs.cmd`` and ``RunFsc.sh``?  Do we really have to shell out via a command shell to invoke a compiler?  It feels like something is flawed if that's the case and using extra shell scripts always brings pain - and especially CMD scripts.  Why can't we just run an executable?